test/ecto/changeset_test.exs

```diff
@@ -874,13 +875,32 @@ defmodule Ecto.ChangesetTest do
     assert changeset.valid?
     assert changeset.errors == []
 
+    # not accepted
     changeset = changeset(%{"terms_of_service" => "false"})
                 |> validate_acceptance(:terms_of_service)
     refute changeset.valid?
     assert changeset.errors == [terms_of_service: {"must be accepted", []}]
 
+    changeset = changeset(%{"terms_of_service" => "other"})
+                |> validate_acceptance(:terms_of_service)
+    refute changeset.valid?
+    assert changeset.errors == [terms_of_service: {"must be accepted", []}]
+
+    # empty params
+    changeset = changeset(%{})
+                |> validate_acceptance(:terms_of_service)
+    refute changeset.valid?
+    assert changeset.errors == [terms_of_service: {"must be accepted", []}]
+
+    # invalid params
+    changeset = changeset(:invalid)
+                |> validate_acceptance(:terms_of_service)
+    refute changeset.valid?
+    assert changeset.errors == []
```

Comment:
doube-checking: is skipping validation fine here, when params is already `:invalid`? this matches behavior of validate_change